docs/manual/format.md

```diff
@@ -229,7 +216,7 @@ In our example there would be 32 such 16-byte index entries, followed
 by the data section:
 
 ```
-00000210: 72 70 6d 00 32 2e 31 2e 32 00 31 00 52 65 64 20    rpm.2.1.2.1.Red 
+00000210: 72 70 6d 00 32 2e 31 2e 32 00 31 00 52 65 64 20    rpm.2.1.2.1.Red
```

Comment:
IMHO The trail space is expected in that particular case

Reply:
Fixed, my editor removed the trailing space